module-2/backend/week_three.md

```diff
@@ -10,14 +10,83 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+<br>
+`rails new project_name`
+<br>
+<br>
+
 2. What do Models generally inherit from in rails?
+<br>
+ActiveRecord
+<br>
+<br>
+
 3. What do Controllers generally inherit from in a rails project?
+<br>
+ApplicationController
+<br>
+<br>
+
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+<br>
+If you only wanted the `index` route for example:
+`resources :horse, only: [:index]`
+or
+`get '/horse' => 'horse#index'`
+<br>
+<br>
+
 5. What rake task is useful when looking at routes, and what information does it give you?
+<br>
+`rake routes` shows you all the routes you have available, as well as the prefix, verb, url patter, and which controller and method you'd need to use for each.
+<br>
+<br>
+
 6. What is an example of a route helper? When would you use them?
+<br>
+A route helper is a rails short cut that helps to link to the various routes without having to manually enter the path. You can use them in your views to link to any of your available routes.
+
+<br>
+<br>
+
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+<br>
+`_url` will return the full url and `_path` will just return the relative path.
+<br>
+<br>
+
 8. What are strong params and why are the necessary?
+<br>
+Strong params are the permissible params that can be passed through our app.
+
+<br>
+<br>
+
 9. What role does `form_for` play in helping us create our forms?
+<br>
+`form_for` is a shortcut that helps us create forms in rails. 
+<br>
+<br>
+
 10. How does `form_for` know where to submit the user's input?
+<br>
+It knows depending on which http verb we're using.
```

Comment:
Interesting! It's true that rails knows what to do with input from a form based on the verb/URI combination, but I think we were one step back from that: how does `form_for` know what to put in the `action` field (and the hidden `method` field for an update) when it generates the form in HTML?

It uses the argument passed to it (e.g. `@horse` in the example you provide below), and uses it to generate a path and determine the proper method to hit the right route to either create or update. Let us know if this makes sense or if you'd like to discuss! 🎉